components/LazyImage.vue

```diff
@@ -51,5 +51,5 @@ export default {
 </script>
 
 <template>
-  <img ref="img" :src="initialSrc" v-bind="$attrs" />
+  <img ref="img" :src="initialSrc" v-bind="$attrs" :alt="altText" />
```

Comment:
The variable altText appears to be undefined wherever its used; is this supposed to fall back to some default value?